pkg/controller/networkpolicy/validate.go

```diff
@@ -404,6 +404,11 @@ func GetAdmissionResponseForErr(err error) *admv1.AdmissionResponse {
 
 // createValidate validates the CREATE events of Antrea-native policies,
 func (v *antreaPolicyValidator) createValidate(curObj interface{}, userInfo authenticationv1.UserInfo) (string, bool) {
+	return v.createAndUpdateValidate(curObj)
+}
+
+// createAndUpdateValidate validates the CREATE and UPDATE events of Antrea-native policies,
+func (v *antreaPolicyValidator) createAndUpdateValidate(curObj interface{}) (string, bool) {
```

Comment:
nit: perhaps name it validatePolicy for comsistency with validateTierForPolicy, validateRuleName ...?

Reply:
Thanks. Done.